File: corpus/2017q1/_p2q3_ok/_check/C-;0620.java

import java.util.Scanner;

public class RafaelCerioniTognato {

    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int n = 0;
        System.out.println("Digite o número de linhas e colunas que desjea:");
        n = teclado.nextInt();
        int A[][] = new int [n][n];
        LeMatriz(A, n);
        int B[][] = new int [n][n];
        MatrizB(B,A,n);
        ImprimeMatriz(A,n);
        ImprimeMatriz(B,n);
    }

    public static void LeMatriz( int m[][], int n){
        Scanner teclado = new Scanner(System.in);
        int lin = 0, col = 0;
        for(lin = 0; lin < n; lin++){
            for(col = 0; col < n; col++){
                System.out.println("Digite o valor que deseja para a matriz A na linha " + lin + " e coluna " + col);
                m[lin][col] = teclado.nextInt();
            }
            col = 0;
        }
    }

    public static void MatrizB (int m[][], int d[][], int n){
        int lin = 0, col = 0, a = n - (n/2);
        for(lin = 0; lin < a; lin++){
            for(col = 0; col < a; col++){
                m[lin][col] = d[lin][col];
            }
            col = 0;
        }
        lin = a;
        col = a;
        for(lin = a; lin < n; lin++){
            for(col = a; col < n; col++){
                m[lin][col] = 0;
            }
            col = 0;
        }
    }

    public static void ImprimeMatriz(int m[][], int n){
        int lin = 0, col = 0;
        System.out.println("Imprimindo a matriz:");
        for(lin = 0; lin < n; lin++){
            for(col = 0; col < n; col++){
                System.out.println("\n" + m[lin][col]);
            }
            col = 0;
        }
    }

}

File: corpus/py/C-;0620.py
int n = 0
print("Digite o número de linhas e colunas que desjea:")
n = int(input())
LeMatriz(A, n)
MatrizB(B,A,n)
ImprimeMatriz(A,n)
ImprimeMatriz(B,n)
def LeMatriz( int m[][], int n):
    int lin = 0, col = 0
    for 0 in range(0, n):
        for 0 in range(0, n):
            print("Digite o valor que deseja para a matriz A na linha " + lin + " e coluna " + col)
        col = 0
def MatrizB(int m[][], int d[][], int n):
    int lin = 0, col = 0, a = n - (n/2)
    for 0 in range(0, a):
        for 0 in range(0, a):
        col = 0
    lin = a
    col = a
    for a in range(a, n):
        for a in range(a, n):
        col = 0
def ImprimeMatriz(int m[][], int n):
    int lin = 0, col = 0
    print("Imprimindo a matriz:")
    for 0 in range(0, n):
        for 0 in range(0, n):
            print("\n" + m[lin][col])
        col = 0